Foundation/FileManager.swift

```diff
@@ -1720,39 +1720,35 @@ open class FileManager : NSObject {
 #if os(Windows)
         NSUnimplemented()
 #else
-        let fd1 = open(file1Rep, O_RDONLY)
-        guard fd1 >= 0 else {
-            return false
-        }
-        defer { close(fd1) }
-
-        let fd2 = open(file2Rep, O_RDONLY)
-        guard fd2 >= 0 else {
-            return false
-        }
-        defer { close(fd2) }
-
-        let buffer1 = UnsafeMutablePointer<UInt8>.allocate(capacity: bufSize)
-        let buffer2 = UnsafeMutablePointer<UInt8>.allocate(capacity: bufSize)
+        guard let file1 = FileHandle(fileSystemRepresentation: file1Rep, flags: O_RDONLY, createMode: 0) else { return false }
+        guard let file2 = FileHandle(fileSystemRepresentation: file2Rep, flags: O_RDONLY, createMode: 0) else { return false }
+        
+        var buffer1 = UnsafeMutablePointer<UInt8>.allocate(capacity: bufSize)
+        var buffer2 = UnsafeMutablePointer<UInt8>.allocate(capacity: bufSize)
         defer {
             buffer1.deallocate()
             buffer2.deallocate()
         }
-
+        
```

Comment:
Bleeding whitespace

Reply:
The space between `defer` and the next likes seems necessary, but If you feel strong about this I will update it.

Also while we are in the topic, I have noticed a couple more unnecessary whitespaces as will in the same method; I will remove them too.

Reply:
Having an empty line is not an issue, its the trailing whitespace which is the problem (the large green rectangle).